samples/playbook_repository.yml

```diff
@@ -0,0 +1,25 @@
+---
+#
+# samples/playbook_repository.yml
+#
+
+# Playbook sample for deploying IBM Spectrum Scale (GPFS) cluster using
+# repository installation method. You will need to provide the URL of an
+# (existing) Spectrum Scale YUM repository. Copy the contents of
+# /usr/lpp/mmfs/{{ scale_version }}/* to a web server to build your repository.
+# A YUM repository will be defined on each managed node in your cluster.
+# Packages will then be installed from this central repository.
+
+# Note that specifying the variable 'scale_version' is mandatory for this
+# installation method.
+
+- hosts: cluster01
+  vars:
+    - scale_version: 5.0.4.0
+    # Remember the trailing slash `/` in the URL
+    - scale_install_repository_url: http://server/path/
```

Comment:
I don't know how this is used, but seems strange that we would require the / at the end, and missing it would cause failure. Rather, how come we do not correctly build up the URL ?

Reply:
@olemyk FYI...

Absolutely right. Whenever using the URL we should assume that users "forget" the trailing slash and add one ourselves (just in case). However, the current code contains something like this:

```
...{{ scale_install_repository_url }}gpfs_rpms...
```

Until all of these are resolved we encourage users to add the trailing slash. You may want to create a ticket for this (improve robustness / make trailing slash optional) so we don't forget.

Reply:
Humm,  yea.. I feel like `{{ scale_install_repository_url }}gpfs_rpms` should be fixed to `{{ scale_install_repository_url }}/gpfs_rpms`  and not rely on the user providing the end slash...  double slash is OK, but missing is fatal error...   

For example, this loads OK https://github.com/IBM/ibm-spectrum-scale-install-infra//pull/237 <-- double // in the URL.

Reply:
Totally agree. I've opened #242 to keep track of this enhancement (so that I don't forget to "fix it later"™). Here, I'm just documenting the current state.